Makefile

```diff
@@ -1,5 +1,8 @@
 init:
-	pip install -r requirements.txt
+	poetry install
```

Comment:
These are all commands you should use to interact with the code now, mentioned in the readme